config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -2656,7 +2712,7 @@ presubmits:
           ../test-infra/hack/bazel.sh build --config=remote --remote_instance_name=projects/k8s-prow-builds/instances/default_instance $(bazel query --keep_going --noshow_progress "kind(.*_binary, rdeps(//... -//vendor/..., //...)) except attr('tags', 'manual', //...)") //build/release-tars
         command:
         - bash
-        image: launcher.gcr.io/google/bazel:0.24.1
+        image: launcher.gcr.io/google/bazel:0.25.2
```

Comment:
The point I was trying to make in mentioning code freeze is that the master jobs already run with this configuration, and for the duration of code freeze master ~= release-1.15, so I feel like this would keep expected behavior consistent rather than alter it.  Have you seen significant differences between master and release-1.15 jobs thus far?

Reply:
Ah thank you. This makes sense!
As for differences: haven’t seen any significant difference between the mater and 1.15 dashboard. 
But thank you for clarifying this.